lib/Dist/Zilla/PluginBundle/Basic.pm

```diff
@@ -45,6 +46,7 @@ It includes the following plugins with their default configuration:
 * L<Dist::Zilla::Plugin::PruneCruft>
 * L<Dist::Zilla::Plugin::ManifestSkip>
 * L<Dist::Zilla::Plugin::MetaYAML>
+* L<Dist::Zilla::Plugin::MetaJSON>
```

Comment:
I was think about something more visible (well, as far as POD for a PluginBundle can be considered visible) such as a block in the DESCRIPTION section:
```pod
B<Note:> L<[MetaCPAN]|Dist::Zilla::Plugin::MetaJSON> is included since Dist::Zilla v6. 
```